Source/Core/Duality/DualityApp.cs

```diff
@@ -526,12 +526,13 @@ public static void ApplyResolution(int width, int height, bool fullscreen)
 		/// <summary>
 		/// Performs a single update cycle.
 		/// </summary>
-		public static void Update()
+		/// <param name="forceFixedStep">If true use a timestep thats equal to <see cref="Time.MsPFMult"/> for the update</param>
+		public static void Update(bool forceFixedStep = false)
```

Comment:
We actually need two different Update overloads for compatibility reasons. While the default parameter allows for source compatibility, it still breaks binary compatibility: A package depending on this method signature would need to be recompiled in order to work again. Using a default parameter value is something that can be done in v3.0, but not here.

Reply:
Didn't realized this would break binary compatibility. Ill change it at once.